priv/templates/extended_bin

```diff
@@ -163,14 +163,22 @@ PIPE_DIR="${PIPE_DIR:-/tmp/erl_pipes/$NAME/}"
 
 # Extract the target cookie
 COOKIE_ARG="$(grep '^-setcookie' "$VMARGS_PATH" || true)"
+DEFAULT_COOKIE_FILE="$HOME/.erlang.cookie"
 if [ -z "$COOKIE_ARG" ]; then
-    echo "vm.args needs to have a -setcookie parameter."
-    exit 1
+    if [ ! -f "$DEFAULT_COOKIE_FILE" ]; then
+        echo "vm.args needs to have a -setcookie, or $DEFAULT_COOKIE_FILE is required."
+        exit 1
+    elif [ -O "$DEFAULT_COOKIE_FILE" ] && [ $(stat --format=%a $DEFAULT_COOKIE_FILE) -eq 400 ]; then
```

Comment:
Yeah, makes sense to check the permissions, but can we broaden the check to just ensure it's readable by the current user, rather than requiring that the file have exactly 400 set on it?